server/routes/api.js

```diff
@@ -132,7 +133,9 @@ export function repo(router) {
       const checkContext = getCheckByType(type).CONTEXT
       if (checkContext) {
         // autobranch doesn't have a context
-        await githubService.protectBranch(repo.json.owner.login, repo.json.name, repo.json.default_branch, checkContext, user.accessToken)
+        await githubService.protectBranch(owner, name, defaultBranch, checkContext, token)
+        const config = new ZapprConfiguration(zapprFile)
+        await checkRunner.runSingle(repo, getCheckByType(type).TYPE, {config: config.getConfiguration(), token})
```

Comment:
I'm not happy with `await` here, because it blocks the user's request unnecessarily. Worst case we probs get a timeout from the ELB. OTOH I can't test it very well if I remove it (e.g. like in `api.test.js` where I want to assert some API calls were made).